social-messenger/src/components/MessagingInput/MessagingInput.css

```diff
@@ -185,3 +185,13 @@ div.rta__autocomplete.str-chat__emojisearch {
   color: rgba(0, 0, 0, 0.9);
   mix-blend-mode: normal;
 }
+
+@media screen and (max-width: 640px) {
+  div.rta__autocomplete.str-chat__emojisearch {
+    width: unset;
+  }
+
+  div.messaging-input__button.emoji-button {
+    display: none;
```

Comment:
is this removing the emoji button in the main message list too? we only want to remove it if the message input is in a thread

Reply:
yes, it is! I didn't realize threads were a thing and missed it in your first comment in Slack. Have updated.